app/Repositories/PhoneSettingsInfoRepositories.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * 
+ * User: Amr Muhamed
+ * Date: 1/5/2019
+ * Time: 2:42 PM
+ */
+
+namespace App\Repositories;
+
+
+use Bosnadev\Repositories\Contracts\RepositoryInterface;
+use Bosnadev\Repositories\Eloquent\Repository;
+
+
+class PhoneSettingsInfoRepositories extends Repository
+{
+
+	/*
+	**********************
+	*	Get Phone with   *
+	*	 primary info    *
+	*	@param $id 		 *
+	**********************
+	*/
+	public function index($primary_id)
+	{
+		$info = $this->findWhere(['contactinfo_id'=>$primary_id],['*']);
+		return $info;
+	}
+
+	/*
+	***************************************
+	*	Function make decision  		  *
+	*	 for save all phone related with  *
+	*		 main settings 			      *
+	*	@param $primary_id 		 		  *
+	*	@param $data 		 		      *
+	***************************************
```

Comment:
invalid phpdoc format, check this as example 

<img width="461" alt="screen shot 2019-01-10 at 11 56 31 am" src="https://user-images.githubusercontent.com/2943218/50960797-dde37280-14ce-11e9-8be6-6b6b89319f25.png">